### PR TITLE
https support

### PR DESCRIPTION
**Pull request by michielbdejong**
ref #4

### CODE REVIEW

File: src/standalone.ts [hunk 1]
```diff
@@ -7,9 +8,25 @@ const aud = process.env.AUD || 'https://localhost:8443'
...
+const tlsDir = process.env.TLS_DIR
+let httpsConfig
+if (tlsDir) {
+  httpsConfig = {
+    key: fs.readFileSync(`${tlsDir}/privkey.pem`),
+    cert: fs.readFileSync(`${tlsDir}/fullchain.pem`)
+  }
+}
+
+const owner = process.env.OWNER || 'https://michielbdejong.inrupt.net/profile/card#me'
```
Comment (jaxoncreed) [inline src/standalone.ts]: It probably should throw an error if you don't have an owner rather than defaulting to your webId.

**Comment by michielbdejong** [comment]
@jaxoncreed please re-review

File: src/server.ts [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
+const OPENID_CONFIG_JSON = fs.readFileSync('./static/openid-configuration.json')
 const DATA_BROWSER_HTML = fs.readFileSync('./static/index.html')
+const MASHLIB_JS = fs.readFileSync('./static/index.html')
+
+type HttpsConfig = {
+  key: Buffer,
+  cert: Buffer
+}
+
+type OptionsObject = {
```
Comment (jaxoncreed) [inline src/server.ts]: This should be an Interface

File: src/server.ts [hunk 3]
```diff
@@ -1,15 +1,33 @@
...
 import { defaultConfiguration } from 'solid-idp'
 
 const debug = Debug('server')
 
+const OPENID_CONFIG_URL_PATH = '/.well-known/openid-configuration'
+const OPENID_CONFIG_JSON = fs.readFileSync('./static/openid-configuration.json')
 const DATA_BROWSER_HTML = fs.readFileSync('./static/index.html')
+const MASHLIB_JS = fs.readFileSync('./static/index.html')
+
+type HttpsConfig = {
```
Comment (jaxoncreed) [inline src/server.ts]: This should be an interface

File: static/openid-configuration.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (jaxoncreed) [inline static/openid-configuration.json]: Why does this file exist? We should be using the openid-configuration generated by solid-idp

**Pull request merged**